samples/gamma/main.c

```diff
@@ -0,0 +1,159 @@
+#include <stdint.h>
+#include <string.h>
+#include <math.h>
+#include <hal/video.h>
+#include <hal/debug.h>
+#include <windows.h>
+
+static void draw_title(const char* title)
+{
+    extern int nextCol;
+    extern int nextRow;
+    nextCol = 25;
+    nextRow = 25;
+    debugPrint("%s", title);
+    for(int i = strlen(title); i < 60; i++) {
```

Comment:
nit: our other samples use space between keyword and open paren in control structures

Reply:
This is actually inconsistent in the existing samples.
I did look at mesh, triangle and led - they all seem to mix these.

Only the new audio sample seems to use spaces consistently.

I personally prefer the non-space style, but am willing to change this if you want me to (waiting for response).

Ideally we'll take care of this with clang-format rules in the future.

Reply:
Hmm.. this is weird. You're right, looks like we mix styles annoyingly. Let's not worry for now and clang-format it in the future.